src/main/rx/rx.h

```diff
@@ -76,6 +76,14 @@ extern const char rcChannelLetters[];
 
 extern int16_t rcData[MAX_SUPPORTED_RC_CHANNEL_COUNT];       // interval [1000;2000]
 
+typedef enum {
+    RX_CHANNEL_STABILITY_LOW = 0,
+    RX_CHANNEL_STABILITY_MEDIUM,
+    RX_CHANNEL_STABILITY_HIGH,
+} rxChannelStability_e;
+
+extern uint8_t rcChannelStability[MAX_SUPPORTED_RC_CHANNEL_COUNT];
```

Comment:
yeah, i thought about using bitmasks but hadn't found the need for them with the current usage scenarios so just went with something simple to start with.